### PR TITLE
Patch legacy article 3

### PR DESCRIPTION
**Pull request by manwar**
Hi @briandfoy 

Please review the PR.
Another set of 10 articles patched.

Many Thanks
Best Regards,
Mohammad S Anwar

### CODE REVIEW

File: content/legacy/_pub_2010_03_more-code-less-cruft-managing-distributions-with-distzilla.md [hunk 1]
```diff
@@ -195,7 +195,7 @@ You can delete your "License and Copyright" sections. `Pod::Weaver` will generat
 
 #### **Release Automation**
 
-Now you're in the home stretch, ready to understand the "maximum overkill" approach to using Dist::Zilla. First, get rid of the version setting in the *dist.ini* and load the [AutoVersion](http://search.cpan.org/perldoc?Dist::Zilla::Plugin::AutoVersion) plugin. It will set a new version per day, or use any other sort of scheme you configure. Then add [NextRelease](http://search.cpan.org/perldoc?Dist::Zilla::Plugin::NextRelease), which will update the changelog with every new release. In other words, the changelog file now starts with:
+Now you're in the home stretch, ready to understand the "maximum overkill" approach to using Dist::Zilla. First, get rid of the version setting in the *dist.ini* and load the [AutoVersion](http://search.cpan.org/perldoc?Dist::Zilla::Plugin::AutoVersion) plugin. It will set a new version per day, or use any other sort of scheme you configure. Then add [NextRelease](https://metacpan.org/pod/Dist::Zilla::Plugin::NextRelease), which will update the changelog with every new release. In other words, the changelog file now starts with:
```
Comment (dnmfarrell) [inline content/legacy/_pub_2010_03_more-code-less-cruft-managing-distributions-with-distzilla.md]: Looks like you missed a search.cpan link here :)

File: content/legacy/_pub_2010_03_more-code-less-cruft-managing-distributions-with-distzilla.md [hunk 2]
```diff
@@ -97,11 +97,11 @@ At this point, you can build a releasable tarball by running `dzil build` (inste
 
 #### **Eliminating Pointless Packaging Files**
 
-The *MANIFEST.SKIP* file tells other packaging tools which files to exclude when building a distribution. You can keep using it (with the [ManifestSkip](http://search.cpan.org/perldoc?Dist::Zilla::Plugin::ManifestSkip) plugin), but you can almost always just drop the file and use the [PruneCruft](http://search.cpan.org/perldoc?Dist::Zilla::Plugin::PruneCruft) plugin instead. It prunes all the files people usually put in their skip file.
+The *MANIFEST.SKIP* file tells other packaging tools which files to exclude when building a distribution. You can keep using it (with the [ManifestSkip](http://search.cpan.org/perldoc?Dist::Zilla::Plugin::ManifestSkip) plugin), but you can almost always just drop the file and use the [PruneCruft](https://metacpan.org/pod/Dist::Zilla::Plugin::PruneCruft) plugin instead. It prunes all the files people usually put in their skip file.
```
Comment (dnmfarrell) [inline content/legacy/_pub_2010_03_more-code-less-cruft-managing-distributions-with-distzilla.md]: Another search.cpan link here

**Comment by dnmfarrell** [comment]
hey @manwar thank you very much for this. Your changes look fine, but I found a couple more search.cpan.org links in the changed files (I've commented on them them). Could you update those two as well please?

**Comment by manwar** [comment]
Hi @dnmfarrell 

Please review the changes.

Many Thanks.
Best Regards,
Mohammad S Anwar

**Comment by dnmfarrell** [comment]
Excellent, thanks!

**Pull request merged**